.buildkite/ftr_configs.yml

```diff
@@ -374,6 +374,7 @@ enabled:
   - x-pack/test/security_functional/expired_session.config.ts
   - x-pack/test/security_solution_endpoint_api_int/config.ts
   - x-pack/test/security_solution_endpoint/config.ts
+  - x-pack/test/security_solution_endpoint_configuration/config.ts
```

Comment:
Looking at these I feel that the split-up config should be moved into a nested folder along with the current one. Something like

```json5
x-pack/test/security_solution_endpoint/endpoint/config.ts
x-pack/test/security_solution_endpoint/integrations/config.ts
```

with `integrations` config for all the tests that were moved (policy, lists, fleet etc). Also move the tests in those folders if you decide to do this.

Another way would be just to have two config files. `security_solution_endpoint/config_endpoint.ts`, `security_solution_endpoint/config_integartions.ts`.

I think some other FTR tests do something similar. `saved_object_api_integration/security_and_spaces`, and `session_view` to name a few.

Reply:
@ashokaditya https://github.com/elastic/kibana/pull/159014/commits/eea7dbd05bf884eaca30d9343f27495ae621ca0e